### PR TITLE
WIP: WPT integration

### PR DESCRIPTION
**Pull request by marcoscaceres**
Work in progress to show WPT results and addition to caniuse. 

cc @sideshowbarker

### CODE REVIEW

File: src/w3c/wpt.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+// Show WPT results
+export const name = "w3c/wpt";
+
+/**
+ *
+ * @param {RespecConfig} conf
+ */
+export async function run(conf) {
+  // if (!conf.wpt) return;
+  // Gett the run ids first
```
Comment (sideshowbarker) [inline src/w3c/wpt.js:10]:
```suggestion
  // Get the run ids first
```

**Review by mera15 — CHANGES_REQUESTED** [review]
profiles/w3c.js
wpt-results

**Review by jhonmurga — COMMENTED** [review]
``

**Comment by marcoscaceres** [comment]
Problem we still have is some features only being exposed on particular platforms/os combinations… see Web Share API, which is on Firefox Windows and Android. Similarly, for Chrome: only supported on Android. 

Not much we can do about that, as wpt.fyi doesn’t test every platform combo… but it makes things a little messy/confusing for somebody that doesn’t know.

**Pull request closed without merging**